pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -336,7 +336,6 @@ func (c *Cloud) ensureLoadBalancerHealthCheck(loadBalancer *elb.LoadBalancerDesc
 	if err != nil {
 		return fmt.Errorf("error configuring load-balancer health-check: %v", err)
 	}
-
```

Comment:
Don't add this - it'll just cause merge conflicts, either for you or for someone else!